src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Ssl.cs

```diff
@@ -133,6 +134,25 @@ internal static SafeSharedX509NameStackHandle SslGetClientCAList(SafeSslHandle s
             return handle;
         }
 
+        internal static bool AddExtraChainCertificates(SafeSslHandle sslContext, X509Chain chain)
+        {
+            Debug.Assert(chain != null, "chain != null");
```

Comment:
For the most part I simply want assert messages to be self-explanatory, since there can be ambiguity.  I'd be fine with ```"Expected chain != null"```, ```"Passed chain == null"```, ```Expected chain != null, got null```, or some other such phrase that just makes it clear if the assert does fire exactly what the problem is.